src/main/java/nomadrealms/game/GameState.java

```diff
@@ -49,6 +49,11 @@ public void update() {
 		world.update(lastInputFrame());
 	}
 
+	public Tile getMouseHexagon(Mouse mouse, Vector2f cameraPosition) {
```

Comment:
Could you make this take in a camera object instead? In the future we might want to add zoom to the camera, which would change the logic